src/main/java/com/staffbase/plugins/sdk/sso/SSOData.java

```diff
@@ -131,72 +135,76 @@
    **********************************************/
 
   /**
-   * The unique id of the specific plugin instance that was requested using
-   * staffbase's SSO.
+   * The name of the audience the staffbase's SSO data is dedicated.
    */
-  private final String instanceID;
+  private final String audience;
+
+  private final String branchID;
```

Comment:
please add documentation comments to `branchID` and `branchSlug` just like the other private fields have them.